### ISSUE TITLE
Idea submission form is broken?

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/hybox-ideas#27]
I submitted an idea here: https://docs.google.com/forms/d/16BiWACwtD4AARGe6P0IaMQl5rOeWDLdi6g6-6V3YGto/viewform?fbzx=-86786976012364307

When it was complete, it said I could see it here: https://github.com/projecthydra-labs/hybox-ideas/issues, but I don't see it.

**Comment by anarchivist** [comment]
Was it #28? The form uses Zapier (I think) and as such only creates new issues every 15 minutes.

**Comment by jcoyne** [comment]
Yes, that was it.  Maybe the success message could be updated to set the expectation that it will _eventually_ show up as an issue.
